schemas/context/profile-contact-crm.schema.json

```diff
@@ -0,0 +1,154 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-contact-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile work details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This mixin is used to capture information regarding person details in context of Business to Business contact.",
+  "definitions": {
+    "profile-contact-crm": {
+      "properties": {
+        "xdm:contactID": {
+          "title": "Contact Record Id",
+          "description": "Contact record id.",
+          "type": "string"
+        },
+        "xdm:sourceType": {
+          "title": "Contact Source Type",
+          "description": "Contact source type.",
+          "type": "string"
+        },		
+        "xdm:ownerID": {
+          "title": "Contact Owner id",
+          "description": "The ID of the owner of the account associated with this contact.",
+          "type": "string"
+        },
+        "xdm:accountID": {
+          "title": "Contact Account id",
+          "description": "ID of the account that’s the parent of this contact.",
+          "type": "string"
+        },
+        "xdm:userOwnerID": {
```

Comment:
What's the reference to this property?

Reply:
This is the "sfdc_user_owner_id" property from the mkt_person entity.

Reply:
Drop it as we already have covered it by ownerID